npc/quests/quests_moscovia.txt

```diff
@@ -8624,10 +8624,10 @@ OnTouch:
 			mes "- You hear a splashing as -";
 			mes "- something gleaming -";
 			mes "- seems to stare at you!! -";
-			next;
+			close2;
 			specialeffect2 EF_FLASHER;
 			percentheal -100,0;
-			close;
+			end;
```

Comment:
i thought that too but my plan was to add these when updating the whole script. Wouldn't it be weird to only have some parts updated?